### DISCUSSION TITLE
Add chompy bird undertile color

### DISCUSSION THREAD

**Discussion by Fox4148** [runelite/runelite#15047] [Ideas]
Visual Improvement while killing chompy birds.
Like the slayer under tile color, but for chompy.
Also maybe adding a killcount tooltip over the bow.

**Comment by Coollegolas** [comment]
This appears to be addressed in https://github.com/runelite/runelite/pull/2309

**Comment by Nightfirecat** [comment]
Can't this be accomplished by just using the NPC Indicators plugin?

**Comment by Spedwards** [comment]
So that PR was closed, but it seems to have addressed this. Came here hoping something was in the works.

@Nightfirecat It can to a degree. There's no way to differentiate between alive and dead chompy birds so when you have a lot, it can be rather confusing.

Here's just one example of that: ![Example](https://i.imgur.com/fG5FeOv.png)

**Comment by SJacobs501** [comment]
@Spedwards I was literally just hunting Chompy birds an hour ago and fixed this issue while I was at it. I made a pull request but in case it doesn't get accepted (or you want it quicker) you can check my commit on how I fixed it.

**Comment by Alexsuperfly** [comment]
the ability to ignore dead npcs was added in 869b1ba1d9c3c1f67e0bf9471750b1fd3082d82e
